fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/HudRenderCallback.java

```diff
@@ -22,10 +22,14 @@
 import net.fabricmc.fabric.api.event.Event;
 import net.fabricmc.fabric.api.event.EventFactory;
 
+/**
+ * @deprecated Use {@link HudLayerRegistrationCallback} instead. For the closest equivalent, register a layer after {@link IdentifiedLayer#SUBTITLES}.
```

Comment:
> For the closest equivalent, register a layer after {@link IdentifiedLayer#SUBTITLES}.

I'm against this note. Devs can make uninformed decisions about fixing this deprecation based on this note